tee_supplicant/src/ree_fs.c

```diff
@@ -255,6 +265,41 @@ static int tee_fs_write(size_t num_params, struct tee_param *params)
 	len = MEMREF_SIZE(params + 1);
 	sz = 0;
 	rc = fs_seek(file, offset, SEEK_SET);
+	if (rc == -EINVAL) {
+		/*
+		 * OP-TEE tries to seek past the end of file. POSIX
+		 * allows this, but Zephyr - does not. We need to fill
+		 * the file with zeroes to the desired position.
+		 */
+		off_t cur_offset;
+		char zero_buf[8];
```

Comment:
nit question: is there some reasons for exactly 8?

Reply:
Nope :)

My only concern is that it should be small enough to fit on the stack.

What are your suggestions?

Reply:
I was just curios, maybe OPTEE usually write 8 byte aligned values or smth and it worth to be mentioned for someone who might change it in future. You can ignore it and have my RB)